pkg/shell/nav.jsx

```diff
@@ -126,7 +126,7 @@ export class CockpitNav extends React.Component {
 
         return (
             <>
-                <SearchInput placeholder={_("Search")} value={this.state.search} onChange={search => this.setState({ search })} onClear={() => this.setState({ search: "" })} className="search" />
+                <SearchInput placeholder={_("Search")} value={this.state.search} onChange={(_, search) => this.setState({ search })} onClear={() => this.setState({ search: "" })} className="search" />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18222-20230124-120118-5255c8fb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)